src/stacked-bar-chart.js

```diff
@@ -144,7 +223,16 @@ class BarChart extends PureComponent {
                                 )
                             })
                         }
-                        { extras.map((item, index) => renderExtra({ item, x, y, index, width, height })) }
+                        { values.map((item, index) => renderDecorator(
```

Comment:
I'm not sure the `values.map` and `renderDecorator` will make any sense. The renderDecorator prop is supposed to return each value along with the necessary information to render for example a circle on each point. Using `values` (as opposed to `series`) removes the information on what dataset it was originally in. I've no idea how using `series` would work right now though. My suggestions to remove decorator for now. Maybe make a note in the README that this component does not yet support it.

Reply:
Good catch. That's actually something I'm still working out right now. Forgot that code was in there in that state (that will teach me to assume that whatever I finished up last night at 11PM is in any condition for public consumption :) )